test_conformance/printf/util_printf.cpp

```diff
@@ -105,6 +106,73 @@ testCase testCaseInt = {
 };
 
 
+//==================================
+
+// long
+
+//==================================
+
+//------------------------------------------------------
+
+// [string] format  | [string] int-data representation |
+
+//------------------------------------------------------
+
+std::vector<printDataGenParameters> printLongGenParameters = {
+
+    //(Minimum) fifteen-wide,default(right)-justified
+
+    { { "%5d" }, "10000000000L" },
```

Comment:
All of these format strings are missing the `l` modifier I think?
```suggestion
    { { "%5ld" }, "10000000000L" },
```

Reply:
Indeed, corrected